opentok/opentok-web-samples
# OPENTOK-49202: Sample apps for Insertable Streams API for JS SDK

Testing instructions for Basic-Video-Transformer:

* check out this branch locally. 
* Navigate to opentok-web-samples/Basic-Video-Transformer
* run `npm install`
* add your credentials to `js/config.js.` 
* run a server locally that can serve the `index.html.` For example, you can use the following command `python3 -m http.server`
* navigate to `localhost:8000` on Chrome and notice that your publisher gets transformed to a black-white filter.
* join `localhost:8000` from another tab. Notice that both publishers have been transformed.

Testing instructions for Basic-Audio-Transformer:

* check out this branch locally. 
* Navigate to opentok-web-samples/Basic-Audio-Transformer
* run `npm install`
* add your credentials to `js/config.js.` 
* run a server locally that can serve the `index.html.` For example, you can use the following command `python3 -m http.server`
* navigate to `localhost:8000` on Chrome.
* It might be easier to join from a different device (still on Chrome) as a second publisher for this one, but it should work on the same device, too. Notice that the audio from the publisher is about half of the volume that you would expect. 

Testing instructions for Basic-Background-Blur:

* check out this branch locally. 
* Navigate to opentok-web-samples/Basic-Background-Blur
* run `npm install`
* add your credentials to `js/config.js.` 
* run a server locally that can serve the `index.html.` For example, you can use the following command `python3 -m http.server`
* navigate to `localhost:8000` on Chrome.
* notice the blur 🙌
